CMakeLists.txt

```diff
@@ -137,7 +137,8 @@ set_option_if(
   ACTS_BUILD_EXAMPLES OR ACTS_BUILD_EVERYTHING)
 set_option_if(ACTS_BUILD_PLUGIN_LEGACY ACTS_BUILD_EVERYTHING)
 set_option_if(ACTS_BUILD_PLUGIN_AUTODIFF ACTS_BUILD_EVERYTHING)
-set_option_if(ACTS_BUILD_EXAMPLES_EXATRKX ACTS_BUILD_PLUGIN_EXATRKX)
+set_option_if(ACTS_BUILD_PLUGIN_EXATRKX ACTS_BUILD_EXAMPLES_EXATRKX)
+set_option_if(ACTS_BUILD_PLUGIN_ONNX ACTS_BUILD_PLUGIN_MLPACK)
```

Comment:
Technically MLPACK is fully independent from ONNX and one could be used without the other. It is just that right now the only example in which it is used need both